blurring_as_a_service/inference_pipeline/components/blur_images.py

```diff
@@ -0,0 +1,119 @@
+import os
+import shutil
+import sys
+
+import cv2
+from azure.ai.ml.constants import AssetTypes
+from cv2.mat_wrapper import Mat
+from mldesigner import Input, Output, command_component
+
+sys.path.append("../../..")
+from blurring_as_a_service.settings.settings import (  # noqa: E402
+    BlurringAsAServiceSettings,
+)
+
+config_path = os.path.abspath(
+    os.path.join(os.path.dirname(__file__), "..", "..", "..", "config.yml")
+)
+aml_experiment_settings = BlurringAsAServiceSettings.set_from_yaml(config_path)[
+    "aml_experiment_details"
+]
+
+
+@command_component(
+    name="blur_images",
+    display_name="Blur images.",
+    environment=f"azureml:{aml_experiment_settings['env_name']}:{aml_experiment_settings['env_version']}",
+    code="../../../",
+)
+def blur_images(
```

Comment:
why is this in components and not in source? 
do we have some rule/indication when to split something into source and components and when to just use components?

Reply:
Mmm some parts could be moved to source indeed, I will refactor it